package.yaml

```diff
@@ -65,8 +65,10 @@ tests:
     - cardano-crypto
     - cardano-crypto-wrapper
     - cardano-ledger
+    - cardano-ledger-test
```

Comment:
this one is because we want to reuse generators here, right?

Reply:
Indeed.